### ISSUE TITLE
No audio 

### ISSUE THREAD

**Issue by Cyberbolt21** [uberhalit/EldenRingFpsUnlockAndMore#28]
hello, I installed the mod, followed all instructions and when i launch the game, patch seems to work, i have 144hz in menu but i then have no audio in game at all. Checked my windows 11 audio settings and updated all my audio drivers but still nothing. In fairness i suspect it could be some issue on my end as ive been have some weird audio issues on my razer headset here and there sometimes. I suspect it could just be that have some razer audio driver issues. But wanted to know if anyone else is experiencing same no audio issue and if there is a fix.

**Comment by uberhalit** [comment]
Hi, 
that sounds like something on your end. The utility isn't patching anything that should affect audio. Please let me know if it issue persists or if you solved it in the meantime.

**Comment by zizhengwu** [comment]
+1. My spec is Windows 11+ RTX 3080 + HDMI Audio + 2160p 120hz HDR. Using this utility causes no audio. I haven't found a fix yet

**Comment by zizhengwu** [comment]
I think I figured out this is an issue of the elden ring game itself. And using this utility makes the game have no sound at startup "consistently". A fix I found is to keep alt tab between the game and the desktop until the audio work properly.

**Comment by bbi-krismorness** [comment]
I have the same problem (with Elden Ring) -- since using this patch, alt tabbing out of full screens seems to break the audio. And then I just go to sound settings and toggle to a different device and back again to get it to work.

**Comment by bbi-krismorness** [comment]
Yep, I have HDMI audio as well (LG C1 screen) -- for some reason it loses the audio. But I don't have the problem after the first time. But once everything was setup, using the applet to launch the game, no problems.
